### ISSUE TITLE
[redesign-2021] - Update Website Template

### ISSUE THREAD

**Issue by chrisguindon** [EclipseFdn/ecdtools.eclipse.org#92]
As previously noted, the new design is available here:
https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/91

**Deliverables:**

- [x] Implement default styles for header & footer
- [x] Default styles for content (h1-h6, p, ul, li, ...)

The footer section of this site will include more than one repeatable section:

- [x] Webinar Series
- [x] Quotes
- [x] Join our Mailing-lit
- [x] Eclipse Featured Footer

Basically, anything that we plan on re-using across every page of the site.

This should be one of our first priorities for this project.

**Comment by ericpoirier** [comment]
The template has been implemented in the redesign.
